src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/SdkDirectoryWorkloadManifestProviderTests.cs

```diff
@@ -605,7 +605,7 @@ public void ItUsesWorkloadSetFromInstallState()
             CreateMockInstallState("8.0.200", 
                 """
                 {
-                    "workloadVersion": "8.0.201"
+                    "workloadSetVersion": "8.0.201"
```

Comment:
As mentioned in `InstallStateContents`, the JSON property should be `workloadVersion`, so these changes in the tests should be reverted.